### PR TITLE
positioning and floating elements project 

### PR DESCRIPTION
**Pull request by hafedEfheij**
greeting,
This is A New York Times article built using only HTML5 and CSS3 . Displaying the use of FlexBox, Grid etc

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+  color: grey;
+}
+
+/* AUTHOR SOCIAL CONTACT */
+.authorName {
+  font-size: 14px;
+  font-weight: bold;
+  text-decoration: none;
+  color: black;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector and selector on line 98 have the same style definition. Group selectors with the same style definition.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+  padding-left: 20px;
+  line-height: 20px;
+  border-left: 1px solid grey;
+}
+
+.lists-container ul {
+  margin-right: 40px;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector and selector on line 433 have the same style definition. Group selectors with the same style definition.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Hafed Efheij** 
Amazing with your first submission for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
Just a couple of details to correct before you can go ahead:

- [x] Group selectors with the same style definition
- [x] Your project should align elements with a combination of all three techniques (float, flex and grid).
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by ebeagusamuel — APPROVED** [review]
# Project approved :100: 
### Congratulations :champagne: 
##### This project is approved. You can merge this branch and move to the next project.
##### Good luck
slack(@ebeagusamuel)

**Pull request merged**